packages/prism-react-renderer/src/components/__tests__/__snapshots__/Highlight.test.tsx.snap

```diff
@@ -1,24 +1,24 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
-exports[`<Highlight /> snapshots renders correctly 1`] = `
+exports[`<Highlight /> > snapshots > renders correctly 1`] = `
 <div>
   <pre
     class="prism-code language-jsx"
-    style="background-color: rgb(42, 39, 52); color: rgb(154, 134, 253);"
+    style="color: #9CDCFE; background-color: #1E1E1E;"
```

Comment:
This thrashing is due to the default theme colors being changed. The overall structure remains.